index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>BEM FIGMA</title>
+</head>
+
+<body>
+
+    <!--Header-->
+
+    <header class="page-header">
+
+        <div class="nav">
+            <div class="nav__logo-links">
+                <h3 class="nav__logo">Medical Core</h3>
+                <ul class="items">
+                    <li class="items__item"><a class="items__icon" href="#"></a>Home</li>
+                    <li class="items__item"><a class="items__icon" href="#"></a>Product</li>
+                    <li class="items__item"><a class="items__icon" href="#"></a>Pricing</li>
+                    <li class="items__item"><a class="items__icon" href="#"></a>Contact</li>
+                </ul>
+            </div>
+            <div class="nav__button">
+                <ul class="items">
+                    <li class="items__item"><a class="item__text" href="#">Login</a></li>
+                    <li class="items__item"><a class="btn" href="#">JOIN US &nbsp; <i class="fa-solid fa-arrow-right"></i></a></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="page-header">
+            <div class="page-header__content">
+                <h5 class="note note--style">Welcome</h5>
+                <h1 class="title title-style title--header">A Great Place to <br> Receive Care</h1>
+                <p class="text text--style">Get your best looking smile now!</p>
+                <div class="page-header__buttons">
+                    <a href="#" class="btn btn--padding">Join Us</a>
+                    <a href="#" class="btn btn--outlined">Learn More</a>
+                </div>
+            </div>
+        </div>
+    </header>
+
+
+    <!--Main-->
+
+    <main class="main-page">
+
+        <section class="expert-page">
+            <div class="expert-page__img"><img src="/img/thumb-concept.svg" class="image" alt="Main doctor"></div>
+            <div class="expert-page__content">
+                <span class="expert-page__line"></span>
+                <h2 class="title">Meet Our Experts</h2>
+                <p class="text">Problems trying to resolve the conflict between <br>
+                    the two major realms of Classical physics: <br>
+                    Newtonian mechanics </p>
+                <a class="items__item items__item--highlighted" href="#">Learn More <i class="fa-solid fa-chevron-right"></i></a>
```

Comment:
`items__item` is positioned outside the Block (items).
`items__item--highlighted` is positioned outside the Block (items).